lammps/job.pbs

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# ---------------------------------------------------------------- #
+#                                                                  #
+#  Example job submission script for Exabyte.io platform           #
+#                                                                  #
+#  Shows resource manager directives for:                          #
+#                                                                  #
+#    1. the name of the job                (-N)                    #
+#    2. the number of nodes to be used     (-l nodes=)             #
+#    3. the number of processors per node  (-l ppn=)               #
+#    4. the walltime in dd:hh:mm:ss format (-l walltime=)          #
+#    5. queue                              (-q) D, OR, OF, SR, SF  #
+#    6. merging standard output and error  (-j oe)                 #
+#    7. email about job abort, begin, end  (-m abe)                #
+#    8. email address to use               (-M)                    #
+#                                                                  #
+#  For more info visit https://docs.mat3ra.com/jobs-cli/overview/  #
+# ---------------------------------------------------------------- #
+
+#PBS -N LAMMPS-TEST
+#PBS -j oe
+#PBS -l nodes=1
+#PBS -l ppn=1
+#PBS -l walltime=00:00:10:00
+#PBS -q D
+#PBS -m abe
+#PBS -M info@exabyte.io
+#
+## Uncomment the line below and put a desired project name, e.g.:
+##      "seminar-default", or "john-projectname".
+## NOTE: this is required when using organizational accounts.
+## more at https://docs.mat3ra.com/jobs-cli/batch-scripts/directives/
+## The job will be charged to the corresponding project.
+## When commented out, the default project for user is assumed.
+##PBS -A <PROJECT_NAME_IN_ACCOUNTING_SYSTEM>
+
+# load module
+module load lammps
+
+# go to the job working directory
+cd $PBS_O_WORKDIR
+
+# run the molecular dynamics calculation
+mpiexec -n $PBS_NP lmp < in.min
```

Comment:
1. Should we use `mpirun` like in all other examples?
2. Do we need to pipe input to an outputfile `> output` or smth like that?

Reply:
No need to pipe output, defaults to log.lammps file. Can be controlled with separate args